hail/python/hail/utils/json.py

```diff
@@ -0,0 +1,31 @@
+import json
```

Comment:
Hrm. This might be superstition, but I feel like naming files in our project with the same name as standard Python packages has caused issues in the past for us. Can we name this `jsonx.py`? We do a similar thing with `httpx.py`.

Reply:
Renamed in [e76ca57](https://github.com/hail-is/hail/pull/11307/commits/e76ca57a3bdfa444019168d79be2cbc2323c0590).